### ISSUE TITLE
Change `fun` to `func` for Swift examples

### ISSUE THREAD

**Issue by FrankSolleveld** [appt-org/appt-website#71]
**URL**
[SwiftUI Accessibility Announcement](https://appt.org/en/docs/swiftui/samples/accessibility-announcement#:~:text=Accessibility%20announcement%20%2D%20SwiftUI&text=By%20setting%20the%20type%20to,this%20case%2C%20create%20an%20AccessibilityNotification.)

**Suggested improvements**
On line 37 of the Swiftui.md file on Accessibility-Announcement a function is declared the wrong way. It should be `func` instead of `fun`. I tried to fix this in a branch and open a PR but I can't contribute to this repository, hence this issue :)

For example:
`fun` should be `func` in all Swift examples

**Screenshots**
If applicable, provide screenshots, they help us understand the problem.
<img width="677" alt="Screenshot 2024-11-11 at 14 28 35" src="https://github.com/user-attachments/assets/2db2d49d-44e0-44e3-9f6b-939af3ba9c52">

**Comment by JJdeGroot** [comment]
Hey @FrankSolleveld thanks for mentioning the typo, I've just fixed it. To make a PR, you can clone the repo on your personal account, and then submit PR from there to our repo hosted on the `appt-org` account.
